src/features/tokens/useTokenBalance.tsx

```diff
@@ -65,3 +91,19 @@ async function fetchTokenBalance(chainId: ChainId, tokenAddress: Address, accoun
     return balance.toString();
   }
 }
+
+export async function fetchListOfERC721TokenId(chainId: ChainId, tokenAddress: Address, accountAddress: Address): Promise<string[]> {
+  logger.debug(`Fetching list of tokenID for account ${accountAddress} on chain ${chainId}`);
+
+  const hypERC721 = getHypErc721Contract(tokenAddress, getProvider(chainId));
```

Comment:
In the case of a native -> synthetic route (i.e. from sending an original NFT from an origin chain to a remote destination chain), the contract won't be a hypERC721. It'll be just an ERC721